autopilot/prefattach_test.go

```diff
@@ -73,29 +73,23 @@ func TestPrefAttachmentSelectEmptyGraph(t *testing.T) {
 		NewNodeID(pub): {},
 	}
 
-	for _, graph := range chanGraphs {
-		success := t.Run(graph.name, func(t1 *testing.T) {
-			graph, err := graph.genFunc(t1)
-			if err != nil {
-				t1.Fatalf("unable to create graph: %v", err)
-			}
+	for _, chanGraph := range chanGraphs {
+		chanGraph := chanGraph
+		graph, err := chanGraph.genFunc(t)
+		require.NoError(t, err)
```

Comment:
nit: just for consistency, you could also add "unable to create graph" here (or alternatively remove elsewhere since it's pretty obvious anyway).